src/simulator/world.py

```diff
@@ -22,7 +23,9 @@ class World:
        runs the main cycle.
     
     Attributes:
-        objects: dictionary of form [object_id -> Object]
+        registry: an instance of ObjectRegistry through which one can access list of all objects,
+        this object will be passed to other objects so that they can register objects of their own to
+        the list of all objects
```

Comment:
So in summary, object! : )

Reply:
Sorry, I didn't get it. What are you suggesting instead of the explanation that I wrote for the registry?